internal/policy.go

```diff
@@ -0,0 +1,35 @@
+/*
+RabbitMQ Messaging Topology Kubernetes Operator
+Copyright 2021 VMware, Inc.
+
+This product is licensed to you under the Mozilla Public License 2.0 license (the "License").  You may not use this product except in compliance with the Mozilla 2.0 License.
+
+This product may include a number of subcomponents with separate copyright notices and license terms. Your use of these subcomponents is subject to the terms and conditions of the subcomponent's license, as noted in the LICENSE file.
+*/
+
+package internal
+
+import (
+	"encoding/json"
+	"fmt"
+	rabbithole "github.com/michaelklishin/rabbit-hole/v2"
+	topologyv1alpha1 "github.com/rabbitmq/messaging-topology-operator/api/v1alpha1"
+)
+
+func GeneratePolicy(p *topologyv1alpha1.Policy) (*rabbithole.Policy, error) {
+	definition := make(map[string]interface{})
+	if p.Spec.Definition != nil {
```

Comment:
It looks like this might be to make the policy_test easier? But I think that test should be representative of how the function is called, i.e. with a non-nil Definition

Reply:
Yeah it's for the test. But I will remove bc it's clearly confusing logic ;)